unity/CAVE/Assets/W3D_Translator/CLI.cs

```diff
@@ -49,10 +49,46 @@ static void Main()
         }
         Debug.Log(story.pprint());
 
-
-
-        // Use enum (ModeType) to get the type - object (mode) contains the data.
-        // This type is an enum so it doesn't make a difference, will elsewhere
+        foreach(EventTrigger e in story.EventRoot) {
```

Comment:
Is a default case necessary for either of the switch statements? Might be nice to throw an error or something?

Reply:
The built in serializer has been good at catching these sort of errors so far so the exception handling will be around `(Story)serializer.Deserialize(reader);`. The xml file will fail against `caveschema.xsd` if an invalid `<xs:choice>` is given anyways. I'll be doing the majority my error handling in the Python script because of this.

It's going to come down to what's a crucial enough error that it's not worth translating a file, e.g. this invalid choice changes how the entire project looks vs defaulting to a base font when the given one fails. I'm not really sure how those lines are going to play out, I'm sure I'll reach out for a zoom at some point to get some second opinions on all that.